src/tracker/tracker_udp.h

```diff
@@ -75,8 +75,8 @@ class TrackerUdp : public SocketDatagram, public TrackerWorker {
   uint64_t            m_connection_id{};
   uint32_t            m_transaction_id{};
 
-  std::unique_ptr<ReadBuffer>  m_read_buffer;
-  std::unique_ptr<WriteBuffer> m_write_buffer;
+  std::optional<ReadBuffer>  m_read_buffer;
+  std::optional<WriteBuffer> m_write_buffer;
```

Comment:
There's no need to 're-allocate' the buffers, that was from before a rewrite so they can just be included directly.

Reply:
No idea what you mean so removed.